### PR TITLE
Add EditorInterfaces and reorder Readme

### PR DESCRIPTION
**Pull request by dlitvakb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -566,7 +566,45 @@ Creating an api key
 blog_space.api_keys.create(name: 'foobar key', description: 'key for foobar mobile app')
 ```
 
-## Validations
+### Editor Interfaces
```
Comment (neonichu) [inline README.md]: > Editing Interface

File: lib/contentful/management/client.rb [hunk 2]
```diff
@@ -132,6 +133,15 @@ def roles
         ClientRoleMethodsFactory.new(self)
       end
 
+      # Allows manipulation of editor interfaces in context of the current client
+      # Allows listing all editor interfaces for client, creating new and finding one by id.
+      # @see _ README for details.
+      #
+      # @return [Contentful::Management::ClientEditorInterfaceMethodsFactory]
+      def editor_interfaces
```
Comment (neonichu) [inline lib/contentful/management/client.rb]: For consistency sake, I would prefer calling the API here `editing_interfaces` as well, on the other hand the endpoint is called '/editor_interfaces'. Maybe @pedropbr can chime in?

File: lib/contentful/management/client.rb [hunk 3]
```diff
@@ -132,6 +133,15 @@ def roles
         ClientRoleMethodsFactory.new(self)
       end
 
+      # Allows manipulation of editor interfaces in context of the current client
```
Comment (neonichu) [inline lib/contentful/management/client.rb]: Should also be "editing interfaces" similar to our API documentation?

**Comment by neonichu** [comment]
Generally looks good to me, but we should sort out "editing" vs. "editor" interfaces. Seems like the API uses "editing" for documentation, but `/editors_interfaces` for the endpoint. A completely new feature should probably not be that confusing regarding names :)

File: lib/contentful/management/client.rb (hunk 3 above)
Comment (dlitvakb) [inline lib/contentful/management/client.rb]: The name of the `sys.type` is Editor Interface, that's why i called it that way

**Comment by dlitvakb** [comment]
Currently waiting on @contentful/product-management to give the 💚 on changes to Editor Interface API and what will be it's final definition

**Comment by dlitvakb** [comment]
EditorInterface it is, updated to singleton endpoint.

@neonichu feel free to review :)

File: lib/contentful/management/client.rb (hunk 2 above)
Comment (dlitvakb) [inline lib/contentful/management/client.rb]: Editor Interface was the final call

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,7 @@
 ### Added
 * Add Roles and Permissions Support
 * Add Headers, Topics and Webhook Name Support
+* Add Editing Interfaces Support
```
Comment (neonichu) [inline CHANGELOG.md]: Canonical name is now "Editor Interfaces" isn't it? Everything else seems to be consistent with that.

File: lib/contentful/management/client.rb (hunk 2 above)
Comment (neonichu) [inline lib/contentful/management/client.rb]: :+1:

**Comment by neonichu** [comment]
You just missed changing one "Editing Interfaces", except for that it's 👌

**Comment by neonichu** [comment]
:shipit:

**Pull request merged**